code/main/edge/edgeResponseManager.brs

```diff
@@ -0,0 +1,82 @@
+' ********************** Copyright 2023 Adobe. All rights reserved. **********************
```

Comment:
Some of these newly added files have 2023 instead of 2024 in license header

Reply:
#199